### PR TITLE
Wrote Tests

### PR DESCRIPTION
**Pull request by El-Potato-Slayer**
Wrote unit and snapshot tests for:
- App.js
- Navbar.js
- Display.js
- ButtonPanel.js
- Button.js

### CODE REVIEW

File: src/tests/Button.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+import { render, cleanup } from '@testing-library/react';
+import Button from '../components/Button';
+import '@testing-library/jest-dom';
```
Comment (eliasecasta) [inline src/tests/Button.test.js:6]: You are not allowed to use this dependency, please remove it from all your tests and migrate them to use the @testing-library/react dependency.

File: src/tests/ButtonPanel.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, cleanup } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import ButtonPanel from '../components/ButtonPanel';
+import '@testing-library/jest-dom';
```
Comment (eliasecasta) [inline src/tests/ButtonPanel.test.js:6]: Migrate to @testing-library/react dependency.

File: src/tests/Display.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom';
```
Comment (eliasecasta) [inline src/tests/Display.test.js:5]: Migrate to @testing-library/react dependency.

File: src/tests/Navbar.test.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom';
```
Comment (eliasecasta) [inline src/tests/Navbar.test.js:5]: Migrate to @testing-library/react dependency.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️

Hello @El-Potato-Slayer,

You are doing really great so far, I will list you some things that need to be improved before moving on:
- [ ] all functionalities are tested using [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/), `@testing-library/react`, see the [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/testing/useReactTestingLibrary.png) from [Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106580952/). (I added a little more description in the comments about this).
- [ ] all functions in calculate.js file are covered with unit tests. (Currently you are not testing your calculate.js file)

Please correct those issues and send another review request, good luck, and well done!

**Review by Elbie-em — APPROVED** [review]
## STATUS: APPROVED ✅ 

Hi @El-Potato-Slayer, you have done a remarkable job in adding negative and positive unit tests for calculate.js as requested. Congratulations, you can proceed.

*All the best and keep up the good work*

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

**Pull request merged**